cfme/tests/services/test_dialog_element_in_catalog.py

```diff
@@ -115,3 +115,608 @@ def test_validate_not_required_dialog_element():
         1692736
     """
     pass
+
+
+@pytest.mark.tier(2)
+def test_dynamically_popullated_text_area__fields_pass_null_to_ansible_tower_templates():
+    """
+
+    Bugzilla:
+        1696474
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/2h
+        testSteps:
+            1. Add dynamic dialog with "text area" field
+            2. Add service catalog with above created dialog
+            3. Order the service
+        expectedResults:
+            1.
+            2.
+            3. Populated "text area" fields should pass correct value  to ansible tower templates
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_dialog_editor_allows_you_to_modify_field_and_exit_without_saving_dialog():
+    """
+
+    Bugzilla:
+        1707961
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/16h
+        testSteps:
+            1. Add dialog
+            2. Edit a dialog
+            3. Edit and save as many fields as many times as you like
+            4. Exit without saving the dialog
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The UI should confirm that you want to exit without saving your dialog
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_set_specific_dates_and_time_in_timepicker_in_service_dialog():
```

Comment:
How about just `test_service_dialog_timepicker`?

Reply:
@john-dupuy I have already started automating test related to dialog and services which I didn't include here in this PR and these test will be my next target.

also made changes Please have a look